feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SearchSessionsFragment.kt

```diff
@@ -82,6 +83,11 @@ class SearchSessionsFragment : DaggerFragment() {
         val groupAdapter = GroupAdapter<ViewHolder<*>>()
         binding.searchSessionRecycler.adapter = groupAdapter
 
+        binding.searchSessionRecycler.setOnApplyWindowInsetsListener { _, insets ->
```

Comment:
Can you use insetter?🙏
https://github.com/DroidKaigi/conference-app-2020/blob/94335ce72e3e5d94518a93daefd23064a79d56c3/feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt#L94